### ISSUE TITLE
FF_PIKOF4OSD target doesn't define fixed-wing motor outputs

### ISSUE THREAD

**Issue by MrBen1909** [iNavFlight/inav#3306]
### Board and Version
FF_PIKOF4OSD 
Version 1.9.1

### Behavior

Built a custom hex for 1.9.1 and have wired it all up. I am currently using crossfire and the motor is wired to S1. But it is responding to input from Pitch and Roll channels. 

Update following discussion on RC Groups. 

```
Originally Posted by stronnag 
It appears that the target only defines multirotor motors, 1 MR servo and fixed wing servos (no FW motors). Which makes is somewhat useless for anything other than tri/quad/hex copters.
```

**Comment by MrBen1909** [comment]
I have tried the replacement target, however I am now getting servo output on motor 4/5 instead of 3/4 of the motors tab with the fixed wing mix. Is this something I have misconfigured or is the target still broken?

**Comment by shellixyz** [comment]
Motor 4/5 ? This board has more than 4 motor outputs ? Could you tell me which motor pad is connected to which MCU pin ? I assumed they were right I only fixed the timers and added outputs for fixed wing.

**Comment by MrBen1909** [comment]
It has an S5 / S6 I do not have the board to hand, will check when I get home. 
[User_manual_PIKO_F4_OSD.pdf](https://github.com/iNavFlight/inav/files/2096015/User_manual_PIKO_F4_OSD.pdf)

**Comment by shellixyz** [comment]
Right, thanks. I don't know why they were commented as RC4/5 in the target I will change that.

**Comment by fiam** [comment]
Fixed by #3333
